services/github/src/api.js

```diff
@@ -321,7 +321,8 @@ builder.declare({
         msg.details = getReleaseDetails(body);
         msg.installationId = installationId;
         publisherKey = PUBLISHERS.RELEASE;
-        msg.tasks_for = 'github-release';
+        msg.tasks_for = GITHUB_TASKS_FOR.RELEASE;
+        msg.action = body.action;
```

Comment:
So just to clarify, this means that there will be an `action` variable passed into the json-e template, right? If so this looks perfect!

Reply:
Oh! that's a good question. Let me add some tests to make sure it reaches the template. Yes, the goal was to have it there so you could use it.

Reply:
upd: the original payload was available inside the templates already as `event` variable. So maybe I don't need to add a separate key here. 
In `.taskcluster.yml` it was possible to access it under `${event.action}` for `tasks_for == release`